single-line-diagram-core/src/test/java/com/powsybl/sld/iidm/TestCase15GraphWithVoltageIndicator.java

```diff
@@ -0,0 +1,118 @@
+        substation = createSubstation(network, "s", "s", Country.FR);
+        vl = createVoltageLevel(substation, "vl1", "vl1", TopologyKind.NODE_BREAKER, 380, 10);
+        createBusBarSection(vl, "bbs1", "bbs1", 0, 1, 1);
+        createBusBarSection(vl, "bbs21", "bbs21", 1, 2, 1);
+        createBusBarSection(vl, "bbs22", "bbs22", 2, 2, 2);
+        createSwitch(vl, "fA", "fA", SwitchKind.BREAKER, false, false, false, 3, 4);
+        createLoad(vl, "loadA", "loadA", "loadA", order++, ConnectablePosition.Direction.TOP, 4, 10, 10);
+        createSwitch(vl, "d1", "d1", SwitchKind.DISCONNECTOR, false, false, false, 0, 3);
+        createSwitch(vl, "d2", "d2", SwitchKind.DISCONNECTOR, false, false, false, 1, 3);
+
+        createSwitch(vl, "fB", "fB", SwitchKind.BREAKER, false, false, false, 5, 6);
+        createLoad(vl, "loadB", "loadB", "loadB", order++, ConnectablePosition.Direction.TOP, 6, 10, 10);
+        createSwitch(vl, "b1", "b1", SwitchKind.DISCONNECTOR, false, false, false, 2, 5);
+        createSwitch(vl, "b2", "b2", SwitchKind.DISCONNECTOR, false, false, false, 0, 5);
+
+        createBusBarSection(vl, "bbs13", "bbs13", 7, 1, 3);
+        createBusBarSection(vl, "bbs23", "bbs23", 8, 2, 3);
+        createLoad(vl, "loadC", "loadC", "loadC", order++, ConnectablePosition.Direction.TOP, 9, 10, 10);
+        createSwitch(vl, "c1", "c1", SwitchKind.BREAKER, false, false, false, 8, 9);
+        createSwitch(vl, "c2", "c2", SwitchKind.BREAKER, false, false, false, 7, 9);
+
+        createSwitch(vl, "link", "link", SwitchKind.BREAKER, false, false, false, 5, 9);
+
+        withBusInfoProvider = new DefaultDiagramLabelProvider(network, componentLibrary, layoutParameters) {
+            @Override
+            public Optional<BusInfo> getBusInfo(BusNode node) {
+                Objects.requireNonNull(node);
+                BusInfo result;
+                if (node.getBusbarIndex() % 2 != 0) {
+                    result = new BusInfo(ComponentTypeName.VOLTAGE_INDICATOR, true, "Top", null,
+                            Side.RIGHT, null);
+                } else {
+                    result = new BusInfo(ComponentTypeName.VOLTAGE_INDICATOR, false, null, "Bottom");
+                }
+                return Optional.of(result);
+            }
+        };
+    }
+
+    @Override
+    protected ResourcesComponentLibrary getResourcesComponentLibrary() {
+        return new ResourcesComponentLibrary("VoltageIndicator", "/ConvergenceLibrary", "/VoltageIndicatorLibrary");
+    }
+
+    @Test
+    public void testBasic() {
+        runTest(new BasicStyleProvider(),  "/TestCase15GraphWithVoltageIndicator.svg");
+    }
+
+    @Test
+    public void testTopological() {
+        runTest(new TopologicalStyleProvider(network), "/TestCase15GraphWithVoltageIndicatorTopological.svg");
+    }
+
+    @Test
+    public void testNominal() {
+        runTest(new NominalVoltageDiagramStyleProvider(network), "/TestCase15GraphWithVoltageIndicatorNominal.svg");
+    }
+
```

Comment:
I think you can remove that test, the result is almost the same than previous one, and we don't lose coverage by removing it.